packages/bioformats/superbuild.cmake

```diff
@@ -28,20 +28,20 @@ if(bf-dir)
     DOWNLOAD_COMMAND "")
   set(EP_SOURCE_DIR "${bf-dir}")
   set(BOOST_VERSION 1.60)
-  message(STATUS "Building Bio-Formats from local directory (${bf-dir})")
+  message(STATUS "Building OME XML C++ from local directory (${bf-dir})")
```

Comment:
This naming choice is certainly in line with the Java `ome-xml.jar` package name.